README.md

```diff
@@ -21,13 +21,21 @@ To install the plugin, run:
 
 ### Commands
 
-- `ceph`: Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
+- `ceph <args>` : Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
 
-- `operator`: run sub-commands against the Rook-Ceph operator
-  -  `restart`: restart the Rook-Ceph operator
+- `rbd <args>` : Call a 'rbd' CLI command with arbitrary args"
 
-- `help`: Output help text
+- `mon-endpoints` : print mon endpoints"
 
+- `operator <subcommands>` : run sub-commands against the Rook-Ceph operator mentioned below
+  - `restart` : restart the Rook-Ceph operator
+  - `<property> <value>` : Set the property in the rook-ceph-operator-config configmap.
+
+- `rook <subcommands>` : run sub-commands mentioned below
+  - `version` : print the version of Rook and the Ceph daemons version from the CephCluster CR status"
+  - `status` : print the phase and/or conditions of every CR in the namespace"
```

Comment:
Printing the status of all CRs seems verbose. What about these options?
```suggestion
  - `status` : print the phase and conditions of the CephCluster CR
  - `status all` : print the phase and conditions of all CRs 
  - `status <CR>` : print the phase and conditions of CRs of a specific type, such as `cephobjectstore`, `cephfilesystem`, etc 
```

Reply:
added.